bucket/tortoisehg.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "5.7.1",
+    "description": "TortoiseHg. A set of graphical tools and a shell extension for the Mercurial distributed revision control system.",
+    "homepage": "https://tortoisehg.bitbucket.io/",
+    "license": "GNU General Public License",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-5.7.1-x64.msi",
+            "hash": "0221163b190641e422144c55a1088f1cf1a104d47c81175ec1597b44bfdb23dd"
+        },
+        "32bit": {
+            "url": "https://www.mercurial-scm.org/release/tortoisehg/windows/tortoisehg-5.7.1-x86.msi",
+            "hash": "02192542a79b8bc4b312f087cd71242ae6a04fe6c47dcf3d6f78e2cd29e725a8"
+        }
+    },
+    "extract_dir": "PFiles\\TortoiseHg",
+    "bin": [
+        "thgw.exe"
+    ],
```

Comment:
Add all relevant shortcuts and bins

Reply:
Added shortcuts and bins - https://github.com/lukesampson/scoop-extras/pull/5987/commits/58189921f751fde217ec817f16b23ad1a0937c6b